app/models/rule.rb

```diff
@@ -33,6 +33,20 @@ class Rule < ApplicationRecord
   has_many :hosts, through: :rule_results, source: :host
   has_many :rule_references_rules, dependent: :delete_all
   has_many :rule_references, through: :rule_references_rules
+  has_many :required_rule_group_rules, class_name: 'RuleGroupRule',
+                                       as: :requiring,
+                                       dependent: :delete_all
+  has_many :conflicting_rule_group_rules, class_name: 'RuleGroupRule',
+                                          as: :conflicting,
+                                          dependent: :delete_all
+  has_many :included_rule_group_rules, class_name: 'RuleGroupRule',
+                                       as: :including,
+                                       dependent: :delete_all
+  has_many :rule_group_rules, dependent: :delete_all
+  has_many :parent_rule_groups, through: :rule_group_rules, source: :rule_group
```

Comment:
While this is semantically correct, I don't think we will ever use it. When we build the tailoring file, we do it from top to bottom and not from rule to profile. You can keep it here as we never know, but I'd suggest to think about this a bit.

Reply:
Yeah, this association is for the parent-child relationship between rule and rule groups so that's why I thought it was necessary but is there a different way for tracking that relationship? 
So if a rule has a rule group as a parent, there should be a row for this in RuleGroupRules with `rule_group_id` and `rule_id` filled out

Reply:
The database linking is fine, but the `has_many` IMO is unnecessary. It's enough to have it from top-to-bottom, but just try to run the import without them, I think it will pass.

Reply:
I'm going to remove this because I agree that it isn't necessary and I don't think it will be in the future. I also tested the import without it and it worked correctly 👍